front/app/api/projects/useAddProject.ts

```diff
@@ -24,8 +25,9 @@ const useAddProject = () => {
       queryClient.invalidateQueries({ queryKey: projectsKeys.lists() });
       queryClient.invalidateQueries({ queryKey: topicsKeys.lists() });
       queryClient.invalidateQueries({ queryKey: areasKeys.lists() });
+      queryClient.invalidateQueries({ queryKey: meKeys.all() });
```

Comment:
'resetMeQuery'? (maybe it was intentional not to use it in these cases? I won't comment on any other instances of `queryClient.invalidateQueries({ queryKey: meKeys.all() });)` :)